Formula/baresip.rb

```diff
@@ -26,21 +26,26 @@ def install
     libre = Formula["libre"]
     librem = Formula["librem"]
     # NOTE: `LIBRE_SO` is a directory but `LIBREM_SO` is a shared library.
-    system "make", "install", "PREFIX=#{prefix}",
-                              "LIBRE_MK=#{libre.opt_share}/re/re.mk",
-                              "LIBRE_INC=#{libre.opt_include}/re",
-                              "LIBRE_SO=#{libre.opt_lib}",
-                              "LIBREM_PATH=#{librem.opt_prefix}",
-                              "LIBREM_SO=#{librem.opt_lib/shared_library("librem")}",
-                              "MOD_AUTODETECT=",
-                              "USE_AVCAPTURE=1",
-                              "USE_COREAUDIO=1",
-                              "USE_G711=1",
-                              "USE_OPENGL=1",
-                              "USE_STDIO=1",
-                              "USE_UUID=1",
-                              "HAVE_GETOPT=1",
-                              "V=1"
+    args = %W[
+      PREFIX=#{prefix}
+      LIBRE_MK=#{libre.opt_share}/re/re.mk
+      LIBRE_INC=#{libre.opt_include}/re
+      LIBRE_SO=#{libre.opt_lib}
+      LIBREM_PATH=#{librem.opt_prefix}
+      LIBREM_SO=#{librem.opt_lib/shared_library("librem")}
+      MOD_AUTODETECT=
+      USE_G711=1
+      USE_OPENGL=1
+      USE_STDIO=1
+      USE_UUID=1
+      HAVE_GETOPT=1
+      V=1
+    ]
+    if OS.mac?
+      args << "USE_AVCAPTURE=1"
+      args << "USE_COREAUDIO=1"
```

Comment:
Not sure if we want to add alternatives on Linux.

Most distros (e.g. Fedora and AUR) have dedicated plugin packages for installing each as plugins. Though, Homebrew doesn't support this style of metapackage.

A heavy feature option would be Debian's https://packages.debian.org/sid/baresip-core which will use dependencies like `alsa-lib`, `jack`, `mosquitto`, `lame`, `mpg123`, `opencore-amr`, `opus`, `libpng`, `libsndfile`, ...

Reply:
Let's wait until there is demand for it